misc/scripts/install-local.sh

```diff
@@ -358,6 +358,19 @@ case "$url" in
 		# Make the directory available for users
 		sudo chown -R "$LOGNAME":"$LOGNAME" . 2>/dev/null
 	;;
+	*.tar*)
+		download "$url"
+		# hash the file
+		hashcheck "${url##*/}"
+		# unzip file
+		sudo tar -xf "${url##*/}" 1>&1 2>/dev/null
+		# cd into it
+		cd ./*/
+		# export srcdir
+		export srcdir="/tmp/pacstall/$PWD"
+		# Make the directory available for users
+		sudo chown -R "$LOGNAME":"$LOGNAME" . 2>/dev/null
+	;;
```

Comment:
Technically this is already done in `*)`, but I prefer it this way. Remove the `tar` and the `cd` from `*)` and we could merge.

Reply:
Why are we adding a separate case for `.tar`? In this method in future we may have to add different cases for `.gz`, `.xz` and other types of `tar`s (which were currently automatically handled by `*)`)